runner/jobserv_runner/handlers/simple.py

```diff
@@ -224,6 +224,18 @@ def _prepare_secrets(self, log):
         secrets = os.path.join(self.run_dir, 'secrets')
         if not os.path.exists(secrets):
             os.mkdir(secrets)  # probably a rebooted run
+
+        # Deal with docker rate-limiting:
+        try:
+            with open(os.path.expanduser('~/.docker/config.json')) as fin:
+                # copy this into container. It may have hub.docker.io read-only
+                # credentials required to avoid rate-limited pulls
+                secret = os.path.join(secrets, 'docker_host_config.json')
+                with open(secret, 'w') as fout:
+                    fout.write(fin.read())
+        except FileNotFoundError:
+            pass  # Okay, not logged into docker
```

Comment:
Maybe. it makes sense to log some message if the config file was not copied to the secrets?

Reply:
force push fixed.